rules/docker-gnmi.mk

```diff
@@ -34,6 +35,7 @@ $(DOCKER_GNMI)_RUN_OPT += -t
 $(DOCKER_GNMI)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 $(DOCKER_GNMI)_RUN_OPT += -v /etc/localtime:/etc/localtime:ro
 $(DOCKER_GNMI)_RUN_OPT += -v /var/run/dbus:/var/run/dbus:rw
+$(DOCKER_GNMI)_RUN_OPT += --privileged --pid=host
```

Comment:
@qiluo-msft , can you sign off on this `privileged` flag being added to the gnmi container?

This is needed for several reasons - including that in this PR, GCU may require to do restart of host services, or services outside of gnmi. Secondly, gnoi APIs are also located within gnmi, and they also need to talk to host services, and do rw operations.

CC - @hdwhdw